src/czar/CzarChunkMap.h

```diff
@@ -64,6 +64,12 @@ class ChunkMapException : public util::Issue {
 /// simultaneously provided they have their own pointers to the maps.
 /// The pointers to the maps are mutex protected to safely allow map updates.
 ///
+/// This class is generated in CzarFamilyMap _read() and _insertIntoMaps(...).
+/// Once generate, the CzarChunkMap created is constant. The new CzarChunkMap
+/// object replaces the old one. This allows access without mutex
+/// protection, which helps when millions of jobs need to be directed
+/// to the proper worker quickly.
+///
```

Comment:
This could be rewritten in a cleaner way:
```
/// Generated by CzarFamilyMap::_read() and CzarFamilyMap::_insertIntoMaps(...).
/// Instances of CzarChunkMap are immutable after construction. Updates replace
/// the entire CzarChunkMap object (atomic swap), so readers can access it
/// without taking a mutex. This enables high-performance, lock-free lookups
/// when routing millions of jobs to the correct worker.
```

Reply:
That's better.